docs/UserGuide.md

```diff
@@ -55,6 +55,40 @@ Note: There are 2 special names when adding transactions - "Self" and "Others".
 
 After a transaction, each payer in the list of portions owes the payee a fraction of the amount, based on their weights in the list of portions.
 
+#### Relevant Transactions
+
+A transaction is only considered **relevant** if it involves you owing or being owed money
+by someone from your contacts list. For that, a transaction must:
+1. Include "Self" in at least the payee or payer: you must be involved in the transaction.
```

Comment:
Grammar: "Self" must be either the payee, a payer, or both

Reply:
Added